source/_integrations/vacuum.markdown

```diff
@@ -45,6 +45,14 @@ Stop the current activity of the vacuum.
 | ---------------------- | -------- | ---------------------------------------------------------------- |
 | `entity_id`            | yes      | Only act on specific vacuum. Use `entity_id: all` to target all. |
 
+### Service `vacuum.toggle`
```

Comment:
This service is deprecated and support will be completely removed in 2024.2. Not sure we should add it now.

Reply:
If we remove it either way, then I wouldn't add that either. I then close the PRa